packages/pyright-internal/src/common/realFileSystem.ts

```diff
@@ -286,71 +301,81 @@ class RealFileSystem implements FileSystem {
         return stat;
     }
 
-    rmdirSync(path: string): void {
+    rmdirSync(uri: Uri): void {
+        const path = this._getNormalizedPath(uri);
         yarnFS.rmdirSync(path);
     }
 
-    unlinkSync(path: string) {
+    unlinkSync(uri: Uri) {
+        const path = this._getNormalizedPath(uri);
         yarnFS.unlinkSync(path);
     }
 
-    realpathSync(path: string) {
+    realpathSync(uri: Uri) {

```

Comment:
do we still need this?

Reply:
The function? Not sure, but I think that would be a separate change. I'd rather not introduce more complexity if I can help it. I'd remove that separately.